mam/v2/mss/mss.c

```diff
@@ -550,21 +549,21 @@ retcode_t mss_create(ialloc *a, mss_t *m, mss_mt_height_t d) {
     err_guard(0 <= d && d <= MAM2_MSS_MAX_D, RC_MAM2_INVALID_ARGUMENT);
 
 #if defined(MAM2_MSS_TRAVERSAL)
-    m->ap = mam_words_alloc(a, MAM2_MSS_MT_AUTH_WORDS(d));
+    m->ap = malloc(sizeof(word_t) * MAM2_MSS_MT_AUTH_WORDS(d));
```

Comment:
Would you consider to use `__WORDSIZE` defined in `<limits.h>` to reimplement `sizeof(word_t)`?

Reply:
`word_t` will probably disappear in a following iteration.